pkg/cloudprovider/providers/azure/azure_util.go

```diff
@@ -44,6 +47,12 @@ const (
 	loadBalancerRuleIDTemplate  = "/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Network/loadBalancers/%s/loadBalancingRules/%s"
 	loadBalancerProbeIDTemplate = "/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Network/loadBalancers/%s/probes/%s"
 	securityRuleIDTemplate      = "/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Network/networkSecurityGroups/%s/securityRules/%s"
+
+	// InternalLoadBalancerNameSuffix is load balancer posfix
+	InternalLoadBalancerNameSuffix = "-internal"
+
+	// nodeLabelRole specifies the role of a node
```

Comment:
I think this nodeLabel is already defined somewhere else. Let's not duplicate it.

Reply:
Well, searched in the code base. Looks there is no centralized place for define node label. I guess, it is fine to leave as it is.

Reply:
Ok thanks